app/renderer/components/styles/global.js

```diff
@@ -210,4 +210,6 @@ globalStyles.color.activeTabDefaultColor = globalStyles.color.chromePrimary
 globalStyles.color.switchBG_on = globalStyles.color.braveOrange
 globalStyles.color.statsGray = globalStyles.color.chromeText
 
+globalStyles.spacing.tabHeight = globalStyles.spacing.tabsToolbarHeight
```

Comment:
Actually `globalStyles.spacing.tabsToolbarHeight` is not yet used anywhere else (as the tabs bar is yet to be refactored with Aphrodite).